src/core/types/chat.rs

```diff
@@ -98,16 +90,16 @@ pub enum NotPrivateChatType {
 #[cfg(test)]
 mod tests {
     use crate::core::types::*;
-    use serde_json::{from_str, to_string};
+    use serde_json::from_str;
 
     #[test]
     fn channel_de() {
         assert_eq!(
```

Comment:
This is better to split `assert` from structure creation:
```rust
let expected = Chat { ... };
let actual = from_str(...);
assert_eq(expected, actual);
```

Reply:
Accepted

Reply:
...but not all test were rewritten

Reply:
Yes, I forgot about `chat::tests`. But I didn't understand what do you mean right away, so now I'm making test for forwarded message. Meet it in the next commit.

Reply:
I meant that `chat::tests` don't use temp variables (expected / actual).